packages/mobile/src/icons/navigator/AccountKey.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import Svg, { Circle, Path } from 'react-native-svg'
+
+function AccountKey() {
```

Comment:
We use PureComponents for our other icons, which seems like a good idea for SVGs like these.
Can you change this to a PureComponent or React.Memo it?

Reply:
I tried doing this but react navigation doesn't like it since it's passed an option. I did `export default React.memo` though in case they are used anywhere else